receiver/nsxtreceiver/metadata.yaml

```diff
@@ -10,11 +10,6 @@ resource_attributes:
   nsxt.node.type:
     description: The type of NSX Node.
     type: string
-    # TODO: Add support for enum in resource attributes
-    # enum:
-    #   - manager
-    #   - host
-    #   - edge
```

Comment:
The resource attribute value is taken from an external API without any validation against the available enum options, so removing this field as not applicable as of now